src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -39,8 +39,8 @@ label {
 }
 
 [type="checkbox"]:checked + label::before {
-  background: $color-white !important;
-  color: $color-primary !important;
+  background: #fff !important;
+  color: #0071bb !important;
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/11999/files#r406826151